frontend/src/components/appointmentsettings/AppointTwo/AppointTwoCardAddDate.tsx

```diff
@@ -27,19 +39,15 @@ export default function AppointTwoCardAddDate(
                 <div className="setTime">
 
                     <TimeSetter
-                        range="start Time"
-                        startTime={startTime}
-                        setStartTime={setStartTime}
-                        endTime={endTime}
-                        setEndTime={setEndTime}
+                        timeInput={startTime}
+                        setTimeInput={setStartTime}
+                        testIfTimeRestrictionAvailable={noTimeRestriction}
```

Comment:
Why do you pass the function here and not the result directly? And i would make the prop "testIfTimeRestrictionAvailable"/"minTime" optional so you can leave it out when you dont need it

Reply:
Done!